src/hooks/useFocusZone.ts

```diff
@@ -36,26 +36,29 @@ export function useFocusZone(
   const disabled = settings?.disabled
   const abortController = React.useRef<AbortController>()
 
-  useEffect(() => {
-    if (
-      containerRef.current instanceof HTMLElement &&
-      (!useActiveDescendant || activeDescendantControlRef.current instanceof HTMLElement)
-    ) {
-      if (!disabled) {
-        const vanillaSettings: FocusZoneSettings = {
-          ...settings,
-          activeDescendantControl: activeDescendantControlRef.current ?? undefined
-        }
-        abortController.current = focusZone(containerRef.current, vanillaSettings)
-        return () => {
+  useEffect(
+    () => {
+      if (
+        containerRef.current instanceof HTMLElement &&
+        (!useActiveDescendant || activeDescendantControlRef.current instanceof HTMLElement)
+      ) {
+        if (!disabled) {
+          const vanillaSettings: FocusZoneSettings = {
+            ...settings,
+            activeDescendantControl: activeDescendantControlRef.current ?? undefined
+          }
+          abortController.current = focusZone(containerRef.current, vanillaSettings)
+          return () => {
+            abortController.current?.abort()
+          }
+        } else {
           abortController.current?.abort()
         }
-      } else {
```

Comment:
Why was this removed?

Reply:
Looks like this is just a whitespace change.  Git diff ended up weird because of it.  The only piece that actually changed here was the deps array

Reply:
Whoops, you're right!